common/lib/capa/capa/templates/choicetext.html

```diff
@@ -59,10 +59,12 @@
         <span id="answer_${id}"></span>
     </fieldset>
         <input class= "choicetextvalue" type="hidden"  name="input_${id}{}" id="input_${id}" value="${value|h}" />
-    
+
     <div class="indicator-container">
     % if input_type == 'checkbox' or not element_checked:
-        <span class="status ${status.classname}" id="status_${id}"></span>
+        <span class="status ${status.classname}" id="status_${id}">
+            <span class="sr">${_(status.classname)}</span>
```

Comment:
https://build.testeng.edx.org/job/edx-platform-python-unittests-pr/24245/testReport/junit/capa.tests.test_input_templates/ChoiceTextGroupTemplateTest/test_grouping_tag/

There is not status available or something like that, here is the error/stacktrace

`<span class="sr">${_(status.classname)}</span>`
`TypeError: 'list' object is not callable`